app/src/main/java/com/pysiak/simple/clock/ui/alarm/AlarmPresenter.java

```diff
@@ -0,0 +1,25 @@
+package com.pysiak.simple.clock.ui.alarm;
+
+/**
+ * Presenter for my Alarm fragment with {@link AlarmView}
```

Comment:
Alarm view presenter.
Remove link - it is not criticaly needed here. Also remove "fragment" word. I have told you about this before. You are working with AlarmView interface here, it could be implemented not in fragment but in Activity or some other class.